src/resources/postcss/category-colors/frontend-category.pcss

```
@@ -0,0 +1,483 @@
+/* -----------------------------------------------------------------------------
+ * Event Day
+ * ----------------------------------------------------------------------------- */
+.tribe-events-calendar-day {
+	/* Event bar with category color indicator */
+	.tribe-events-calendar-day__event {
+		position: relative;
+
+		&:after {
+			background-color: var(--tec-color-category-primary);
+			content: '';
+			height: 100%;
+			left: calc(var(--tec-grid-width-1-of-9) - var(--tec-grid-gutter-half));
+			position: absolute;
+			width: 4px;
+		}
+	}
+
+	/* Featured event styling */
+	.tribe-events-calendar-day__event--featured {
+		.tribe-events-calendar-day__event-datetime-featured-icon-svg,
+		.tribe-events-calendar-day__event-datetime-featured-text {
+			color: var(--tec-color-category-primary);
+		}
+
+		&:after {
+			background-color: var(--tec-color-category-primary);
+		}
+	}
+
+	/* Category pill container */
+	.tec-events-calendar-day__event-categories {
+		display: inline;
+
+		/* Individual category pill */
+		.tec-events-calendar-day__category {
+			align-items: center;
+			background-color: var(--tec-color-category-secondary);
+			border-radius: var(--tec-events-border-radius-pill);
+			color: var(--tec-color-category-text);
+			display: flex;
+			font-size: var(--tec-font-size-2);
+			line-height: var(--tec-line-height-3);
+			padding: 0 var(--tec-spacer-1);
+			width: fit-content;
+
+			/* Category icon */
+			.tec-events-calendar-day__category-icon {
+				aspect-ratio: 1/1;
+				background-color: var(--tec-color-category-primary);
+				border-radius: var(--tec-events-border-radius-circle);
+				display: inline-block;
+				margin-right: var(--tec-spacer-0);
+				width: 14px;
+			}
+		}
+	}
+}
+
+/* -----------------------------------------------------------------------------
+ * Event Month
+ * ----------------------------------------------------------------------------- */
+.tribe-events {
+	/* Calendar event styling */
+	.tribe-events-calendar-month__calendar-event {
+		.tribe-events-calendar-month__calendar-event-details {
+			background-color: var(--tec-category-secondary);
+			border-left: 3px solid var(--tec-category-primary);
+			color: var(--tec-category-text);
+			padding: var(--tec-spacer-1);
+		}
+
+		.tribe-events-calendar-month__calendar-event-datetime {
+			color: var(--tec-color-category-text);
```

I think this should just override the existing prop, rather than creating a new one:
```suggestion
			--tec-color-text-event-date: var(--tec-color-category-text);
```

---

Similar all around, but that is going to cause a big delay at this point.

---

I tested this approach, and while overwriting the props isn't overly bad doing something like

```
[class*="tribe_events_cat-"] {
}
```

However, it would require some massaging because I don't want to overwrite every prop across the board. So I agree, maybe in the future we could make that change. It would condense the CSS a lot.